recipes/robotframework-interpreter/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "robotframework-interpreter" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66323c3cc577800a3df08acd0ff642567fcd3f028ad995f816186824b7b305c6
+
+build:
+  number: 0
```

Comment:
This can probably be `noarch: python`

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python